internal/cli/runner_agent.go

```diff
@@ -129,7 +135,14 @@ func (c *RunnerAgentCommand) Run(args []string) int {
 }
 
 func (c *RunnerAgentCommand) Flags() *flag.Sets {
-	return c.flagSet(0, nil)
+	return c.flagSet(flagSetOperation, func(set *flag.Sets) {
+		f := set.NewSet("Command Options")
+		f.BoolVar(&flag.BoolVar{
+			Name:   "enable-dynamic-config",
```

Comment:
Maybe without `enable`?

```suggestion
			Name:   "dynamic-config",
```

Reply:
So I considered that but I prefer to have verbs in my bool flags because it helps folks understand what the value-less expression (the most common one for a bool flag) is.

Reply:
Ah, makes sense! Thanks for explanation :)